internal/services/privatednsresolver/private_dns_resolver_inbound_endpoint_resource.go

```diff
@@ -80,7 +81,7 @@ func (r PrivateDNSResolverInboundEndpointResource) Arguments() map[string]*plugi
 					"private_ip_address": {
 						Type:     pluginsdk.TypeString,
 						Optional: true,
-						Computed: true,
+						Computed: !features.FourPointOhBeta(),
```

Comment:
Based on your info we can revert this
```suggestion
						Computed: true,
```